utils/pythonutils/utils.go

```diff
@@ -152,6 +158,50 @@ func getFilePath(srcPath, fileName string) (string, error) {
 	return filePath, nil
 }
 
+// Create the CmdOutputPattern objects that can capture group content that may span multiple lines for logs that have line size limitations.
+// Since the log parser parse line by line, we need to create a parser that can capture group content that may span multiple lines.
+func getMultilineSplitCaptureOutputPattern(startCollectingPattern, captureGroup, endCollectingPattern string, handler func(pattern *gofrogcmd.CmdOutputPattern) (string, error)) (parsers []*gofrogcmd.CmdOutputPattern) {
+	// Prepare regex patterns.
+	oneLineRegex := regexp.MustCompile(startCollectingPattern + `(` + captureGroup + `)` + endCollectingPattern)
+	startCollectionRegexp := regexp.MustCompile(startCollectingPattern)
+	endCollectionRegexp := regexp.MustCompile(endCollectingPattern)
+
+	// Create a parser for single line pattern matches.
+	parsers = append(parsers, &gofrogcmd.CmdOutputPattern{RegExp: oneLineRegex, ExecFunc: handler})
+
+	// Create a parser for multi line pattern matches.
+	lineBuffer := ""
+	collectingMultiLineValue := false
+	parsers = append(parsers, &gofrogcmd.CmdOutputPattern{RegExp: regexp.MustCompile(".*"), ExecFunc: func(pattern *gofrogcmd.CmdOutputPattern) (string, error) {
```

Comment:
the parser that we use goes over the content line by line so It will not include newlines anyway since it is the delimiter used for tokens.
The purpose of this parser is to remove any newlines and concatenate the text removing new lines that was added if split